django_socio_grpc/proto_serializers.py

```diff
@@ -78,6 +78,14 @@ def data_to_message(self, data):
         assert hasattr(
             self.Meta, "proto_class"
         ), f'Class {self.__class__.__name__} missing "Meta.proto_class" attribute'
+
+        # Choice value -> Enum key
```

Comment:
It would be hard to limitate more than that, if you take an IntegerChoices("Mychoice", ["ONE", "TWO"]) as exemple, to use Mychoice.ONE, the serializer field needs the value 1 while the grpc needs the value "ONE", so we always needs to invert the value and the name of the enum member when doing serializer <---> message